projects/dsp-ui/src/lib/viewer/values/time-value/time-input/time-input.component.html

```diff
@@ -15,4 +15,10 @@
             <span class="custom-error-message">Time should be given in precision HH:MM.</span>
         </mat-error>
     </mat-form-field>
+    <div class="date-form-error">
+        <mat-error *ngIf="((dateFormControl.hasError('validDateTimeRequired') || timeFormControl.hasError('validDateTimeRequired')) &&
```

Comment:
Maybe it would be easier to understand this if the null checks come first.

Reply:
done in a210b88